signer/src/transaction_coordinator.rs

```diff
@@ -806,6 +818,24 @@ where
                 sig: Vec::new(),
             };
 
+            let msg_public_key = msg.signer_pub_key;
+
+            let sender_is_coordinator =
+                given_key_is_coordinator(msg_public_key, bitcoin_chain_tip, &signer_set);
+
+            let public_keys = &coordinator_state_machine.get_config().signer_public_keys;
+            let public_key_point = p256k1::point::Point::from(msg_public_key);
+
+            // check that messages were signed by correct key
+            if !Self::authenticate_message(
+                &packet,
+                public_keys,
+                public_key_point,
+                sender_is_coordinator,
+            ) {
```

Comment:
I would break this into multiple lines like so:
```rust
let is_authenticated = Self::authenticate_message(
    &packet,
    public_keys,
    public_key_point,
    sender_is_coordinator,
);

if !is_authenticated {
    ...
```
I find it a bit easier to read

Reply:
Done